src/assets/i18n/pt-PT.json5

```diff
@@ -242,9 +248,18 @@
   // "admin.registries.metadata.schemas.no-items": "No metadata schemas to show.",
   "admin.registries.metadata.schemas.no-items": "Nenhum esquema de metadados para mostrar.",
 
+  // "admin.registries.metadata.schemas.select": "Select",
+  "admin.registries.metadata.schemas.select": "Selecionar",
```

Comment:
Portuguese from Portugal is "Seleccionar" instead of "Selecionar" which is the Portuguese from Brasil.

Reply:
@rsaraivac please ignore my suggestion I wasn't considering the AO